software/excalidraw.yml

```
@@ -0,0 +1,10 @@
+name: excalidraw
+website_url: https://excalidraw.com
+description: Virtual whiteboard for sketching hand-drawn like diagrams
+licenses:
+  - MIT
+platforms:
+  - Typescript
+tags:
+  - Miscellaneous
+source_code_url: https://github.com/excalidraw/excalidraw
```

> https://docs.excalidraw.com/docs#try-now

As the main website is more of a demo rather than docs, it may be a better fit as a demo with the main site being occupied by the documentation.

```suggestion
  - Miscellaneous
source_code_url: https://github.com/excalidraw/excalidraw
demo_url: https://excalidraw.com
```